components/teams-service/server/v2/server.go

```diff
@@ -142,55 +142,27 @@ func (s *Server) AddTeamMembers(ctx context.Context,
 		return nil, status.Error(codes.InvalidArgument, "missing user IDs")
 	}
 
-	// verify team exists and isn't filtered out by the project filter
-	_, err := s.service.Storage.GetTeam(ctx, req.Id)
-	if err != nil {
-		return nil, service.ParseStorageError(err, req.Id, "team")
-	}
-
-	_, err = s.service.Storage.AddUsers(ctx, req.Id, req.UserIds)
+	updatedUserIDs, err := s.service.Storage.AddUsers(ctx, req.Id, req.UserIds)
```

Comment:
the goal of this PR: simplify this server function to one single storage call that does all the work: 
- checking team existence within the project filter
- adding (or removing) the given user IDs
- returning the final updated list of user IDs